### PR TITLE
[RELEASE] v3.0.0

### PR DESCRIPTION
**Pull request by MbNakaya**
## Story
https://moippagamentos.atlassian.net/browse/DX-215

## Description
Release SDK to **v3.0.0**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,33 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+<a name="v3.0.0"></a>
+## v3.0.0 (2018-01-04)
+
+
+#### Bug Fixes
+
+*   Fixed errors treatment ((3862cc69))
```
Comment (caiogaspar) [inline CHANGELOG.md]: Todos os hash de commit dessa versão e da anterior (2.2.0) estão sem links, ficando fora do padrão.

File: src/Resource/Holder.php [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+        $phone = $this->getIfSet('phone', $response);
+
+        $holder->data->phone->countryCode = $this->getIfSet('countryCode', $phone);
+        $holder->data->phone->areaCode = $this->getIfSet('areaCode', $phone);
+        $holder->data->phone->number = $this->getIfSet('number', $phone);
+        $holder->data->birthDate = $this->getIfSet('birthDate', $response);
+        $holder->data->taxDocument = new stdClass();
+        $holder->data->taxDocument->type = $this->getIfSet('type', $this->getIfSet('taxDocument', $response));
+        $holder->data->taxDocument->number = $this->getIfSet('number', $this->getIfSet('taxDocument', $response));
+        //$holder->data->addresses = [];
```
Comment (caiogaspar) [inline src/Resource/Holder.php]: Remove esse comentário.

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,33 @@ All notable changes to this project will be documented in this file.
...
+#### Tests
+
+*  Support PHPUnit 5 and 6 ((5a316192))
+*  Test against PHP 7.1 and 7.2 ((c6983f63))
+* **Order**
+  *  add tests for the new mapped attributes ((9c99d832))
+  *  add attribute paid to order creation mock ((2f01ad91))
+
+#### BREAKING CHANGES
+The type `Holder` was added to be used on payment creation and at some other features linked on funding instrument.
```
Comment (somentelucas) [inline CHANGELOG.md]: * and **on** some other features

File: README.md [hunk 4]
```diff
@@ -255,13 +256,23 @@ $orders = $this->moip->orders()->getList(new Pagination(10,0), $filters, 'josé
...
 
+##### Inserir dados do portador
+Para realizar o pagamento, via cartão de crédito, utilizando o cartão de um terceiro (quando o cliente não é o portador do cartão que será utilizado), é necessário que estes dados sejam diferenciados e informados corretamente, para cada etapa do fluxo.
+
+```php
+$holder = $this->moip->holders()->setFullname('Jose Silva')
+    ->setBirthDate(\DateTime::createFromFormat($this->date_format, $this->date_string))
+    ->setTaxDocument('22222222222', 'CPF')
+    ->setPhone(11, 66778899, 55)
+    ->setAddress(Holder::ADDRESS_BILLING, 'Avenida Faria Lima', '2927', 'Itaim', 'Sao Paulo', 'SP', '01234000', '8');
```
Comment (somentelucas) [inline README.md]: Tá confuso o que é `complement`... Ao invés de colocar `'8'` coloca `Apt 101`

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,33 @@ All notable changes to this project will be documented in this file.
...
+
+*   Fixed errors treatment ((3862cc69))
+
+#### Features
+
+* **Holder**  Add Holder resource and resolves Holder params. ((9f84f793))
+* **Order**
+  *  map attributes on order creation ((8f9755ef))
+  *  add attribute paid to order creation mock ((2f01ad91))
+  *  remove shipping address attributes from populate ((7eda9b6f))
```
Comment (somentelucas) [inline CHANGELOG.md]: I think you are also returning the `escrow` now, right? Is it something that should be here?

File: CHANGELOG.md (hunk 1 above)
Comment (MbNakaya) [reply CHANGELOG.md]: Done

File: src/Resource/Holder.php (hunk 2 above)
Comment (MbNakaya) [reply src/Resource/Holder.php]: Done

File: CHANGELOG.md (hunk 3 above)
Comment (MbNakaya) [reply CHANGELOG.md]: Done

File: README.md (hunk 4 above)
Comment (MbNakaya) [reply README.md]: Done

File: CHANGELOG.md (hunk 5 above)
Comment (MbNakaya) [reply CHANGELOG.md]: No, the escrow was added recently to SDK-Java. The SDK-PHP has the escrow resource a few months ago.

**Pull request merged**